cdap-app-templates/cdap-etl/cdap-data-streams/src/main/java/co/cask/cdap/datastreams/DataStreamsSparkLauncher.java

```diff
@@ -92,6 +104,46 @@ public void initialize() throws Exception {
       sparkConf.setMaster(String.format("local[%d]", numSources + 1));
     }
     context.setSparkConf(sparkConf);
+
+    if (!checkpointsDisabled) {
```

Comment:
because the default behavior is to enable checkpoints. The default value of a boolean is false, so if the property was 'checkpointEnabled', the absence of the property would imply checkpointEnabled=false.

that being said, it only matters in the app config. I can flip it here if its really a concern, but thought it would be better to be consistent with whats in the config.